core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4086,16 +4100,33 @@ public synchronized HttpRedirect doQuietDown(boolean block, int timeout) {
      *
      * @param block Block until the system really quiets down and no builds are running
      * @param timeout If non-zero, only block up to the specified number of milliseconds
-     * @param reason Quiet reason that will be visible to user
-     * @since 2.267
+     * @param message Quiet reason that will be visible to user
+     * @deprecated since TODO; use {@link #doQuietDown2(boolean, int, String, boolean)} instead.
      */
-    @RequirePOST
+    @Deprecated
     public HttpRedirect doQuietDown(@QueryParameter boolean block,
                                     @QueryParameter int timeout,
-                                    @QueryParameter @CheckForNull String reason) throws InterruptedException, IOException {
+                                    @QueryParameter @CheckForNull String message) throws InterruptedException, IOException {
+
+        return doQuietDown2(block, timeout, message, false);
+    }
+
+    /**
+     * Quiet down Jenkins - preparation for a restart
+     *
+     * @param block Block until the system really quiets down and no builds are running
+     * @param timeout If non-zero, only block up to the specified number of milliseconds
+     * @param message Quiet reason that will be visible to user
+     * @param safeRestart If the quietDown is for a safeRestart
+     * @since TODO
+     */
+    @RequirePOST
+    public HttpRedirect doQuietDown2(@QueryParameter boolean block,
```

Comment:
We don't need all of the `…2` method names here, Stapler prioritized dispatch to non-`@Deprecated` methods.

Reply:
I tried this but it won't compile anymore without any meaningful exception ("Compilation failed")

Reply:
The error is

```
[ERROR] error: javax.annotation.processing.FilerException: Attempt to reopen a file for path …/core/target/classes/jenkins/model/Jenkins/doQuietDown.stapler
```

Remove the Stapler-related annotations (`@QueryParameter`) from the deprecated overload and it should work.